### ISSUE TITLE
Network driver returns 404s for Discover notifications

### ISSUE THREAD

**Issue by tomdee** [projectcalico/libnetwork-plugin#14]
See https://github.com/docker/libnetwork/blob/master/docs/remote.md#discovernew-notification

I'm not sure it matters...

**Comment by robbrockbank** [comment]
I'm planning on taking this on as part of libnetwork ramp-up.  Should get to it early next week.

**Comment by robbrockbank** [comment]
Suggestion from @tomdee Maybe need discovery parm

**Comment by tomdee** [comment]
Just tried this and I got this in the docker logs
```
DEBU[0129] discovery notification error : Plugin Error: NetworkDriver.DiscoverNew, {
  "Err": "404: Not Found"
} 
```

I started docker with
```
sudo docker daemon -D --cluster-store etcd://127.0.0.1:2379 --cluster-advertise=127.0.0.1:9999
```

**Comment by tomdee** [comment]
Fixed by #17
